app/views/users/_user.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class='userContainer'>
+  <div class='userImageContainer'>
+    <img class='userImage' src=<%= user.photo %> alt='userImage' />
+  </div> 
+  <div class='userInformationContainer'>
+    <h1><%= user.name %></h1>
+    <p>Number of posts: <%= user.post_counter %></p>
+</div>
```

Comment:
OPTIONAL

- [ ] I'm required to change the routes manually in the browser before I can navigate through the pages of your app 🤔.
 
- [ ] It would be nice if you add all the necessary elements to enable users to navigate from one page to another improving the user's experience of your app💪

e.g (This will allow you to navigate to a particular user)

```ruby
<a href="/users/<%= user.id %>">
    <div><%= user.name %></div>
</a>
```

Reply:
Great recomendation @abdulhamiid, thanks !!